src/components/WhimsicalInstaller/WhimsicalInstaller.helpers.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import { random } from '../../utils';
+
+export type Point = { x: number, y: number };
+export type BezierPath = {
+  startPoint: Point,
+  endPoint: Point,
+  controlPoint: Point,
+};
+
+export type FileStatus =
+  | 'autonomous' // Flying autonomously towards the file
+  | 'being-inhaled' // Very close to the folder, being sucked in
+  | 'swallowed' // At the very center of the folder, no longer active
```

Comment:
Perhaps `being-captured` and `captured` for these two?

Reply:
Yeah, I could do that. I like the quirkiness of inhaling/swallowing in this context, since the folder kinda moves like a mouth. But capturing is more descriptive, since captured files can't be interacted with.

When providing suggestions like this, it'd be awesome if you could add a brief bit about your rationale :) In this case I agree with you, but in cases where the benefits of a change are less clear, it'd be helpful to have a sense of why you're proposing a change.

Reply:
Because I have dependent PRs on this one, and those other PRs continue with the current metaphor, I think I'll create a new, final PR that does this (along with a couple other cleanup things I want to do)

Reply:
> When providing suggestions like this, it'd be awesome if you could add a brief bit about your rationale :)

I tend to write less rather than more as a habit, but I’ll definitely work on explaining better!